pom.xml

```diff
@@ -4,32 +4,206 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.25</version>
-        <relativePath />
+        <version>3.28</version>
+        <relativePath/>
     </parent>
     <artifactId>audit-log-plugin</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>hpi</packaging>
-    <properties>
-        <jenkins.version>2.60.3</jenkins.version>
-	<java.level>8</java.level>
-    </properties>
+
+    <!-- jenkins project information -->
     <name>Audit Log Plugin</name>
     <description>Audit logging for Jenkins with predefined audit events</description>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/TODO+Plugin</url>
+    <url>https://github.com/jvz/audit-log-plugin</url>
     <licenses>
         <license>
             <name>MIT License</name>
             <url>http://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
-    <!-- Assuming you want to host on @jenkinsci:
-    <scm>
-        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-    </scm>
-    -->
+    <!-- jenkins project global properties -->
+    <properties>
+        <!-- common properties to the project -->
+        <jenkins.version>2.60.3</jenkins.version>
+        <java.level>8</java.level>
+        <commons-lang3.version>3.8.1</commons-lang3.version>
+        <!-- log4j-specific properties -->
+        <log4j.version>2.11.1</log4j.version>
+        <log4j-audit.version>1.0.0</log4j-audit.version>
+        <!-- maven enforcer plugin properties -->
+        <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
```

Comment:
You don't need to redefine all these versions starting from this line to the end of the properties. They're already defined by the parent pom.

Reply:
the log4j-audit version is not defined in the parent pom, removing it from this pom throws errors.

Reply:
I said staring from this line to the end. The log4j-audit version is above this line.

Reply:
lol.. oh okay